README.md

```diff
@@ -2,7 +2,15 @@
 
 A webpack plugin for running single-spa microfrontends in standalone mode. This is an alternative to using [import-map-overrides](https://github.com/joeldenning/import-map-overrides).
 
-_Running in standalone mode is not 100% equivalent to when it's fully integrated into the rest of the app. For this reason, using import-map-overrides on an integrated environment is generally recommended._
+> **⚠️ Warning: Experimental**
+>
+> This plugin should be considered "experimental" in that the core team has not yet settled on a consensus as to if 'standalone mode' is how local development should be approached. Generally speaking, developers want this because "that's the way we used to do it". **There are potential pitfalls that you should understand before deciding to use this plugin.**
```

Comment:
> developers want this because "that's the way we used to do it"

This is a bit pointed / sharp, and sort of blames the developer for wanting something, which I think is a bit too strong. Maybe this?

```suggestion
> This plugin should be considered "experimental" since it is new and is not equivalent to developing in integrated mode. The single-spa core team created this plugin because it's an often-requested feature, but we're not sure if it will result in a better experience for our users.. **There are potential pitfalls that you should understand before deciding to use this plugin.**
```

Reply:
I don't see how it blames developers but rather explains the source of the mentality of the request. My experience is that the devs asking for this are beginners to single-spa who want to hold on to old paradigms. But I'll try to reword it.  Does this reach a middle point?

```suggestion
> This plugin should be considered "experimental" since it is new. The single-spa core team created this plugin because it's an often-requested feature, usually as a hold-over from old paradigms. But we're not sure if it will result in a better experience for developers. **There are potential pitfalls that you should understand before deciding to use this plugin.**
```

Reply:
> as a hold-over from old paradigms

I would contend that running in standalone mode is not an old paradigm. One of the things that convinced me that this standalone plugin is needed is that module federation fully encourages standalone mode as the default development mechanism, and module federation will perhaps soon overtake single-spa as the dominant microfrontends solution.

In other words, I don't think that running a web app locally as a standalone app is an old paradigm, since the new paradigm (module federation) encourages it.

Reply:
> I don't see how it blames developers

The quotes around `"that's the way we used to do it"` come off as air quotes that are perhaps condescending.